src/saveToCase.js

```diff
@@ -568,6 +692,25 @@ define([
                         return;
                     }
                 }
+                
+                var attachmentRegex = /\/case\/attachment\/(\w+)\/@src$/,
+                    attachRet = path.match(attachmentRegex);
+                if (attachRet) {
+                    basePath = path.replace(attachmentRegex, "");
+                    mug = form.getMugByPath(basePath);
+                    if (mug && mug.__className === "SaveToCase") {
+                        var attachProperties = mug.p.attachmentProperty,
+                            nodeName = attachRet[1];
+                        if (!attachProperties) {
+                            attachProperties = {};
```

Comment:
This line creates an object that is only bound to a local name, correct? Is the purpose of this to prevent the next `if` statement from throwing a type error? Even if it does, does it matter that `attachProperties[nodeName] = {};` will not modify `mug.p.attachmentProperty` if this branch is evaluated?